packages/tspd/src/gen-extern-signatures/components/decorators-signatures.tsx

```diff
@@ -77,14 +78,31 @@ export function generateSignatures(
   const context = createTspdContext(program);
   const base = namespaceName === "" ? "__global__" : namespaceName;
   const $decoratorsRef = refkey();
+  const locations = [...new Set(decorators.map((d) => d.location ?? "."))];
+  if (locations.length > 1) {
```

Comment:
So if the same decorator is exposed through multiple subpaths, we just reject it? This is probably okay for now, but some libraries like HTTP Streams start with `import "../lib.tsp"` (i.e. load the root path first) because they need to use types from the base library. The experimental Azure Core decorators don't need this, but many libraries likely will.